### PR TITLE
Create & delete customer

### LINKED ISSUE #6: Create customer
Using the mutation query
```graphql
mutation {
  createUser(friendlyName:"", phoneNumber:"123456789") {
    id
  }
}
```
Build a create customer page that adds customers.

### LINKED ISSUE #11: Delete customer
Use GQL query 

```
mutation deleteCustomer{deleteCustomer(id: "%s"){
				id
			  }}
```

create a "delete user" button on the modify user (single user details page) that deletes the user. This button is not shown on the create a user page.

### PR DESCRIPTION
**Pull request by HuiFang56**

### CODE REVIEW

**Comment by HuiFang56** [comment]
This part works now, but I have got two warnings. I might need some help from you to solve them.

File: client/src/App.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ import "bootstrap/dist/css/bootstrap.css";
 
 import { Route, BrowserRouter as Router, Switch } from "react-router-dom";
 
-import ManageCustomer from "./components/ManageCustomer";
-import ModifyCustomer from "./components/ModifyCustomer";
+import CustomerCreate from "./components/CustomerCreate";
```
Comment (can-z) [inline client/src/App.js]: create is a verb and you want to name this with a noun. For example `CustomerCreateForm`

**Comment by can-z** [comment]
since you merged the create customer PR into this one you need to update the PR title.

**Comment by can-z** [comment]
I tested with this branch, a new customer is not created when I click on the submit button.

**Comment by can-z** [comment]
the placeholder of phone number says `647xxxxxxx`, why do we suggest that the phone number always starts with 647?

File: client/src/App.js (hunk 1 above)
Comment (HuiFang56) [reply client/src/App.js]: I agree, it also indicates this component should contain a form. I just changed the name to `CustomerCreateForm`.

**Comment by HuiFang56** [comment]
> since you merged the create customer PR into this one you need to update the PR title.

I just changed the PR title to `Create & delete customer`.

**Comment by HuiFang56** [comment]
> I tested with this branch, a new customer is not created when I click on the submit button.

Sorry, it happened because I tested diffrent ways to redirect the page after creating a new customer, and I forgot to change it back. I just fixed it on the new commit, you can check it now.

**Comment by HuiFang56** [comment]
> the placeholder of phone number says `647xxxxxxx`, why do we suggest that the phone number always starts with 647?

I just modifed the text on the placeholder to "e.g. +1(xxx)xxx-xxxx". Does this make a sence to you?

**Comment by can-z** [comment]
We currently do not have any rules around that input and so we don't need to restrict the users what they can put in.

**Review by can-z — APPROVED** [review]

**Comment by HuiFang56** [comment]
> We currently do not have any rules around that input and so we don't need to restrict the users what they can put in.

Ok, I will leave the placeholders blank for now.

**Comment by can-z** [comment]
> > We currently do not have any rules around that input and so we don't need to restrict the users what they can put in.
> 
> 
> 
> Ok, I will leave the placeholders blank for now.

You said you were gonna leave it blank but merged the PR without that change, why?

**Pull request merged**